config/keys.js

```diff
@@ -0,0 +1,5 @@
+if (process.env.NODE_ENV === 'production') {
+  module.exports = require('./prod');
```

Comment:
also i don't think you need this file, it's so small it can be in the same file as the server, or what usually happens in this case that they would have a separate file for starting the server, and in there they would have this code and coonection to the database

Reply:
We really need it

Reply:
Done